internal/proto/local/controller/api/services/v1/auth_method_service.proto

```diff
@@ -211,8 +210,12 @@ message AuthenticateRequest {
 }
 
 message AuthenticateResponse {
-  resources.authtokens.v1.AuthToken item = 1;
-  string token_type = 2 [json_name="token_type"];
+  reserved 1, 2; // Old item and token_type
```

Comment:
nit: Since we only expose this API through JSON, I don't think we need to reserve the numbers yet, but not really important.

Reply:
`buf` complains if these aren't marked as reserved since they were previously used.